src/Main.java

```diff
@@ -134,4 +134,22 @@ static void drawEqual(BufferedImage img, int offset) {
         g2d.fillRect(9 + offset, 89, 82, 2);
     }
 
+    static void EquationResult(String[] args) {
+        String result[] = new String[110];
+        int index = 0;
+        for (int i = 0; i <= 9; i++) {

```

Comment:
< 10 would look nicer, but it's up to you, same line below

Reply:
Done as suggested. 
Change:
```
for (int j = 0; j <= 9; j++) {
    if (i + j <= 9) {
```
to:
```
for (int j = 0; j < 10; j++) {
    if (i + j < 10) {
```